exporter/metric/metric_test.go

```diff
@@ -39,14 +40,20 @@ var (
 )
 
 func TestDescToMetricType(t *testing.T) {
+	cloudMock := cloudmock.NewCloudMock()
+	client := cloudMock.MetricServiceClient
 	inMe := []*metricExporter{
 		{
-			o: &options{},
+			o:         &options{},
+			cloudMock: cloudmock.NewCloudMock(),
```

Comment:
You have just created a mock above. Why do you need a new mock for every exporter here?